docker-local/dfsp_b/dfsp_b_thirdparty_scheme_adapter.json

```diff
@@ -0,0 +1,71 @@
+{
+  "INBOUND": {
+    "PORT": 4005,
+    "HOST": "0.0.0.0"
+  },
+  "OUTBOUND": {
+    "PORT": 4006,
+    "HOST": "0.0.0.0"
+  },
+  "WSO2_AUTH": {
+    "staticToken": "0706c62f-c022-3c42-8d14-8b5df128876d",
+    "tokenEndpoint": "",
+    "clientKey": "test-client-key",
+    "clientSecret": "test-client-secret",
+    "refreshSeconds": 3600
+  },
+  "REDIS": {
+    "PORT": 6379,
```

Comment:
I think we should have a separate Redis instance (different port) for every (sdk|thirdparty)-scheme-adapter. Am I wrong?

Reply:
this value isn't with sync with docker-compose file

Reply:
I don't think they need different ports if the hostnames i.e (pisp-redis/dfspa-redis/dfspb-redis) each adapter is pointed to is correct. I believe each redis instance gets their own IP in the docker container.

6379 is the default port of redis I believe, so there shouldn't be a need to specify that under that image.

Do I think that for example the PISP sdk-scheme-adapter and thirdparty-scheme-adapter should have seperate redis instances? Maybe...I'm not sure.

Maybe I'm assuming docker does a lot of magic...

Believe this addresses this comment and a couple following ones.

Reply:
what do you think @lewisdaly - you've reached `magician` level at docker :-D

Reply:
@eoln I want to go back in time to me 5 years ago struggling with docker and say "Don't worry young me, one day it will all make sense."

Reply:
Yeah so long as the hostnames are different, there's no need to have a different port. The only reason we would want a different port is if we wanted to talk to these redis instances _outside_ of docker.